src/test/java/com/rackspace/jenkins_nodepool/Mocks.java

```diff
@@ -100,8 +102,11 @@
     List<Attempt> attemptListSuccess;
     Attempt attemptSuccess;
 
+
     public Mocks() {
         requestTimeout = 30;
+        installTimeout = 60;
+        maxAttempts = 3;
```

Comment:
Should we also allow maxAttempts to be ``-1`` so that it can be set to retry forever?

Reply:
I don't think that would be good, I think we should pass failures back up to the job, so that things don't sit for ever. We can increase the number or retries through the UI if builds are failing too quickly. 

But again this is the mocks class, only used for unit tests.